packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -340,9 +344,15 @@ async function getWalletAddressPage(
   pagination?: Pagination,
   sortOrder?: SortOrder
 ): Promise<WalletAddress[]> {
-  return await WalletAddress.query(deps.knex)
-    .getPage(pagination, sortOrder)
-    .withGraphFetched('asset')
+  const addresses = await WalletAddress.query(deps.knex).getPage(
```

Comment:
Would chaining `.withGraphFetched('asset')` work here as well?

Reply:
It would, but then it won't be using the cached `asset`.